run.sh

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env bash
```

Comment:
A helper script to run the backup, not intended for development not for production use